code/_onclick/hud/hud.dm

```diff
@@ -235,6 +235,7 @@ GLOBAL_LIST_INIT(available_ui_styles, list(
 	floor_change = null
 	hand_slots.Cut()
 
+	QDEL_LIST(open_containers)
```

Comment:
This will literally delete everything within the open container when you close the HUD? This should null the list not delete everything inside of it